manager/volume.go

```diff
@@ -614,7 +617,59 @@ func (m *VolumeManager) Expand(volumeName string, size int64) (v *longhorn.Volum
 	}
 
 	if v.Spec.Size > size {
-		return nil, fmt.Errorf("cannot expand volume %v with current size %v to a smaller size %v", v.Name, v.Spec.Size, size)
+		return nil, fmt.Errorf("longhorn volume %v spec size %v is greater than requested Persistent Volume Claim size %v", v.Name, v.Spec.Size, size)
+
+	}
+
+	if v.Status.ActualSize > size {
```

Comment:
Why we need to check status.ActualSize? I assume we only need to tackle spec.Size.

Reply:
removed.